astropy/cosmology/flrw/tests/test_base.py

```diff
@@ -766,7 +773,7 @@ def test_clone_change_param(self, cosmo):
         assert c.__class__ == cosmo.__class__
         assert c.name == cosmo.name + " (modified)"
         assert c.H0.value == 100
-        for n in set(cosmo.__parameters__) - {"H0"}:
+        for n in set(cosmo.parameters) - {"H0"}:
```

Comment:
`.items()`? and then skip `H0`?

Reply:
Is there anything wrong with this? It's a fair bit shorter and also clear IMO.

Reply:
Yes, just as clear. I just liked the aspect of `.parameters` being a dict, so one can get values out easily.

Reply:
I added a helper function to `conftest` that allows for the `items` and nice filtering. Best of both worlds.